src/openrct2/windows/guest_list.c

```diff
@@ -141,7 +141,7 @@ static sint32 _window_guest_list_highlighted_index; // 0x00F1EE10
 static sint32 _window_guest_list_selected_tab;      // 0x00F1EE12
 static sint32 _window_guest_list_selected_filter;   // 0x00F1EE06
 static sint32 _window_guest_list_selected_page;     // 0x00F1EE07
-static uint32 _window_guest_list_selected_view;  // 0x00F1EE13
+static uint32 _window_guest_list_selected_view;		// 0x00F1EE13
```

Comment:
You aligned this line using tabs, while the others are aligned using spaces. AFAIC you can either change this line to using spaces or change the others to tabs (which is our standard for C files), but the latter option would probably be IntelOrca's preferred solution.